integration-test/build.gradle.kts

```diff
@@ -46,9 +46,7 @@ kotlin {
     wasmWasi {
         nodejs()
     }
-    jvm {
-        withJava()
-    }
+    jvm()
```

Comment:
integration-test uses `withJava` to verify that Kapt works correctly with kotlinx-serialization. Can you recommend how to check it instead?

Reply:
Restored this call to `withJava()` as without it Kapt stops working until Kotlin 2.1.20-Beta2 release. And this brings up the non-smooth user migration path - so I decided to lift error to warning for `withJava()` method usage.